src/composables/trade/useSor.ts

```diff
@@ -352,14 +352,17 @@ export default function useSor({
 
       console.log('[SOR Manager] swapExactIn');
 
-      const swapReturn: SorReturn = await sorManager.getBestSwap(
+      const swapReturn = await sorManager.getBestSwap(
         tokenInAddress,
         tokenOutAddress,
         tokenInDecimals,
         tokenOutDecimals,
         SwapTypes.SwapExactIn,
         tokenInAmountScaled
       );
+      if (!swapReturn) {
+        return;
+      }
```

Comment:
I'm not sure this is a good idea? Thrown errors render as alerts to users and are sent to Sentry, if we just return here no one will know that something failed.

Reply:
I put in try/catch only calculating swapAmount, so  if there will be any error in `sorManager.getBestSwap` except calculating swapAmount, this error will occur in this function as well and error will be sent to sentry